packages/firebase_remote_config/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -13,43 +11,34 @@ part of firebase_remote_config;
 // ignore: prefer_mixin
 class RemoteConfig extends FirebasePluginPlatform with ChangeNotifier {
   RemoteConfig._({this.app})
-      : super(app.name, 'plugins.flutter.io/firebase_remote_config');
+      : super(app!.name, 'plugins.flutter.io/firebase_remote_config');
```

Comment:
This shouldn't use `!` as `app` should be optional.

Reply:
fix -> app? .name ?? ''